configserver/src/main/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandler.java

```diff
@@ -86,7 +88,7 @@ public HttpResponse handleDELETE(HttpRequest request) {
 
         if (isReindexingRequest(request)) {
             applicationRepository.modifyReindexing(applicationId, reindexing -> reindexing.enabled(false));
-            return new JSONResponse(Response.Status.OK);
+            return new JSONResponse(Response.Status.OK) { { object.setString("message", "Reindexing disabled"); } };
```

Comment:
This feels sufficiently magical that I think it'd be good to encapsulate it in a `createJsonResponseWithMessage`-ish method and only do the wizardry once there

Reply:
You're right.